django_inertia/templatetags/inertia_tags.py

```diff
@@ -7,5 +7,5 @@
 
 @register.simple_tag(takes_context=True)
 def inertia(context, app_id="app"):
-    page = context["page"]
+    page = context["__page"]
```

Comment:
Same here, the value should not be hard-coded

Reply:
Thanks for the review and done